src/components/common/sky/sky.module.scss

```diff
@@ -2,12 +2,6 @@
 .sky {
   position: relative;
   height: 100%;
-  background: radial-gradient(
-    circle at 50% 50%,
-    #0d121e 0%,
-    transparent 90%,
-    transparent 100%
-  );
```

Comment:
We cant remove this, is used in the home. Maybe adding a flag for this gradient might be the solution.